### PR TITLE
Refactor monad to more accurate definition

### PR DESCRIPTION
**Pull request by MattMcFarland**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -411,17 +411,7 @@ compose(foo, identity) ≍ compose(identity, foo) ≍ foo
...
-```js
-['cat,dog', 'fish,bird'].chain((a) => a.split(',')) // ['cat', 'dog', 'fish', 'bird']
-
-//Contrast to map
-['cat,dog', 'fish,bird'].map((a) => a.split(',')) // [['cat', 'dog'], ['fish', 'bird']]
-```
-
-`of` is also known as `return` in other functional languages.
-`chain` is also known as `flatmap` and `bind` in other languages.
+A monad is just a monoid in the category of endofunctors, what's the problem?
```
Comment (hemanth) [inline readme.md]: 7d0SSrflqE7Re

**Comment by hemanth** [comment]
7d0SSrflqE7Re

Thank you!

**Comment by jethrolarson** [comment]
The repo is getting big enough for satirical PRs. You've made it @hemanth!

**Comment by MattMcFarland** [comment]
xD Sorry I couldnt help myself.. its been a running joke @work for a few weeks.

**Comment by jethrolarson** [comment]
You're only permitted to make that joke if you understand why it's true ;)

**Pull request closed without merging**